src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image-container">
+        <img 
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Imac A1419"
+        >
+      </div>
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__review">
+        <div class="card__review-left">
+          <div class="card__review-star"></div>
+          <div class="card__review-star"></div>
+          <div class="card__review-star"></div>
+          <div class="card__review-star"></div>
+          <div class="card__review-star"></div>
+        </div>
```

Comment:
Need to fix classes and add the modifier classes for highlight stars
<img width="762" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/d53604a1-a6ad-4e7a-aa4b-5871ea9f961b">
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```